homework1/thirdTask.js

```diff
@@ -0,0 +1,7 @@
+// It reverses number
+
+const currentMaxValue = 4589;
+let reverseMaxValue = +currentMaxValue.toString().split('').reverse().join("");
```

Comment:
Все працює вірно, єдине що я рекомендую всюди використовувати явне перетворення:
```suggestion
let reverseMaxValue = Number(currentMaxValue.toString().split('').reverse().join(""));
```